File: loader/src/main/java/com/blackmorse/hattrick/promotions/model/PromoteType.java
package com.blackmorse.hattrick.promotions.model;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public enum PromoteType {
    AUTO(0, "auto"),
    QUALIFY(1, "qualify");

    private int value;
    private String stringValue;
}

File: akka-loader/src/main/scala/loadergraph/promotions/PromoteType.scala
package loadergraph.promotions

trait PromoteType {
  def value: Int
  def valueStr: String
}

case object Auto extends PromoteType {
  val value = 0
  val valueStr = "auto"
}

case object Qualify extends PromoteType {
  val value = 1
  val valueStr = "qualify"
}




